.circleci/config.yml

```diff
@@ -0,0 +1,30 @@
+version: 2.1
+
+orbs:
+
+workflows:
+  version: 2
+  pipeline:
+    jobs:
+      - build
+
+jobs:
+  build:
+    docker:
+      - image: circleci/node:10.16
+    working_directory: ~/build
+    steps:
+      - checkout
+      - restore_cache:
+          keys:
+            - v1-dependencies-{{ checksum "package.json" }}
+            # fallback to using the latest cache if no exact match is found
+            - v1-dependencies-
+      - run: yarn install
+      - save_cache:
+          paths:
+            - node_modules
+          key: v1-dependencies-{{ checksum "package.json" }}
+      - run: yarn global add typescript 
```

Comment:
Could we use the `npm` client rather than `yarn`? We're using package-locks rather than yarn lockfiles

Reply:
ok will update